netstat.c

```diff
@@ -2070,6 +2071,11 @@ int main
 #endif
     getroute_init();		/* Set up AF routing support */
 
+    path_proc = getenv("PATH_PROC");
```

Comment:
is that some kind of convention standard env for this? If its net tools specific, maybe have a NETTOOLS_PROCPATH instead?